vignettes/release-announcement.Rmd

```diff
@@ -0,0 +1,131 @@
+```{r}
+#| include: false
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're excited to announce that an initial version of the S7 package is now on CRAN.
+S7 is a new OOP system designed to be a successor to S3 and S4.
+The overarching goal of S7 is to combine the best parts of S3 and S4, so that the R community can eventually standardise on a single OOP system.
```

Comment:
I would prefer a narrative more like (a) S3 has proven to be a very useful system for single dispatch, and S4 explored formal class definitions and multiple dispatch as an alternative to S3 (b) but this led to fragmentation where developers were forced to choose between the simpler and more base compatible S3 and the more sophisticated and complicated S4. (c) Therefore, we aim to unify the OOP landscape by implementing the most useful S4 features, along with some new ones, on top of the existing S3 dispatch machinery and through a simpler syntax than S4.

Reply:
Ok, I've done my best to incorporate that narrative into the intro. Let me know what you think.

Reply:
Maybe replacing  _designed to be_   by   _meant to become_  would be more cautious.
It's ok to have the goal here (and be considerably less presumptious (? wording ?) in the  #308